src/components/ConfigSelect/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react'
+
+import { 
+  ConfigLabelText,
+  ConfigLableContainer,
+  ConfigMetricWrapper,
+  SwitchContainer,
+  Picker
+} from '../Globals/ConfigMetric';
+
+import ConfigSelectProps from 'src/interfaces/ConfigSelectProps';
+
+const ConfigSelect = ({
+  label,
+  unit,
+  pickerItems
+}: ConfigSelectProps ) => {
+  const [selectedValue, setSelectedValue] = useState("");
```

Comment:
this can have an internal state too, but we need a way for it to pass its state upwards, because we will need to gather all the config and send it